src/test/resources/features/attachments.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Adding attachments
```

Comment:
@insookwa  lets try to handle each feature file with its corresponding steps and pages in separate pr, personally i think that will make reviewing much easier , cc @kaweesi  whats your opinion on this